app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -163,23 +162,9 @@ AcuantContext.displayName = 'AcuantContext';
 /**
  * Returns a found AcuantJavascriptWebSdk
  * object, if one is available.
- * Depending on the SDK version,
- * will use either startWorkers (11.8.2) or start (11.9.1)
  */
 const getActualAcuantJavascriptWebSdk = (): AcuantJavascriptWebSdkInterface => {
-  if (
-    window.AcuantJavascriptWebSdk &&
-    typeof window.AcuantJavascriptWebSdk.startWorkers === 'function' &&
-    typeof window.AcuantJavascriptWebSdk.start !== 'function'
-  ) {
-    return {
-      ...window.AcuantJavascriptWebSdk,
-      start(...args) {
-        window.AcuantJavascriptWebSdk.startWorkers?.(...args);
-      },
-    };
-  }
-  if (window.AcuantJavascriptWebSdk && typeof window.AcuantJavascriptWebSdk.start === 'function') {
+  if (window.AcuantJavascriptWebSdk) {
     return window.AcuantJavascriptWebSdk;
   }
```

Comment:
```typescript
const getActualAcuantJavascriptWebSdk = (): AcuantJavascriptWebSdkInterface => {
  if (window.AcuantJavascriptWebSdk) {
    return window.AcuantJavascriptWebSdk;
  }
  if (!window.AcuantJavascriptWebSdk) {
    // eslint-disable-next-line no-console
    console.error('AcuantJavascriptWebSdk is not defined in the global scope');
  }
  return window.AcuantJavascriptWebSdk;
};
```

may be simplified?

```typescript
const getActualAcuantJavascriptWebSdk = (): AcuantJavascriptWebSdkInterface => {
  if (!window.AcuantJavascriptWebSdk) {
    // eslint-disable-next-line no-console
    console.error('AcuantJavascriptWebSdk is not defined in the global scope');
  }
  return window.AcuantJavascriptWebSdk;
};
```

Reply:
Thanks for catching that!  I made the change in [f3ea184](https://github.com/18F/identity-idp/pull/10582/commits/f3ea184f7ee5b8e5998587d555d5584803fda787).